code/modules/paperwork/photography.dm

```diff
@@ -452,18 +465,23 @@ GLOBAL_LIST_INIT(SpookyGhosts, list("ghost","shade","shade2","ghost-narsie","hor
 ******************/
 /obj/item/camera/digital
 	name = "digital camera"
-	desc = "A digital camera. A small screen shows there is space for 10 photos left."
+	desc = "A digital camera."
+	digital = TRUE
 	var/list/datum/picture/saved_pictures = list()
 	pictures_left = 30
 	var/max_storage = 10
 
-/obj/item/camera/digital/afterattack(atom/target as mob|obj|turf|area, mob/user as mob, flag)
-	if(!on || !pictures_left || ismob(target.loc)) return
-	captureimage(target, user, flag)
+/obj/item/camera/digital/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>A small screen shows that there are currently [length(saved_pictures)] pictures stored.</span>"
 
+/obj/item/camera/digital/afterattack(atom/target, mob/user, flag)
+	if(!on || !pictures_left || ismob(target.loc))
+		return
+
+	captureimage(target, user, flag)
 	playsound(loc, pick('sound/items/polaroid1.ogg', 'sound/items/polaroid2.ogg'), 75, 1, -3)
 
-	desc = "A digital camera. A small screen shows that there are currently [saved_pictures.len] pictures stored."
 	icon_state = icon_off
 	on = FALSE
 	spawn(64)
```

Comment:
Just noticed this, this should be converted to a callback instead of using `spawn()` here